src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -483,6 +483,7 @@ private function addOpenApiSection(ArrayNodeDefinition $rootNode): void
                                 ->scalarNode('email')->defaultNull()->info('The email address of the contact person/organization. MUST be in the format of an email address.')->end()
                             ->end()
                         ->end()
+                        ->booleanNode('backward_compatibility')->defaultTrue()->info('Enable this to decorate the "api_platform.swagger.normalizer.documentation" instead of decorating the OpenAPI factory.')->end()
```

Comment:
Why not using `canBeEnabled`?

Reply:
because we don't need it?

Reply:
Oh yes it's because it's only a boolean value I see. `canBeEnabled` is used when there are other parameters too.